apps/posts/src/views/post-analytics/components/Newsletter.tsx

```
@@ -0,0 +1,104 @@
+        });
+    };
+
+    return (
+        <div className='grid grow grid-cols-[auto_300px] gap-8 py-5'>
+            <div className='border-t border-border'>
+                <Content />
+            </div>
+            <div className='-mt-px flex basis-[300px] flex-col'>
+                <StatsTabs>
+                    {tabs.map(group => (
+                        <StatsTabsGroup>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)